src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.imp.BetDaoImpl;
+import core.basesyntax.dao.imp.UserDaoImpl;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    private BetDao betDao;
+    private UserDao userDao;
+
+    public ConsoleHandler() {
+        betDao = new BetDaoImpl();
+        userDao = new UserDaoImpl();
+    }
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            System.out.println("Enter name and password:");
+            String userCommand = scanner.nextLine();
+            if (userCommand.equalsIgnoreCase("end")) {
+                return;
+            }
+            User user;
+            try {
+                String[] userData = userCommand.split(" ");
+                String name = userData[0];
+                String password = userData[1];
+                user = new User(name, password);
+            } catch (NumberFormatException | IndexOutOfBoundsException e) {
+                System.out.println("The entered data is incorrect");
+                continue;
+            }
+            userDao.add(user);
+            System.out.println(user);
+            while (true) {
```

Comment:
let's not create a loop inside a loop
try passing everything you need for both models in one line